community/github/pull-requests.md

```diff
@@ -0,0 +1,211 @@
+may want to force commit that change and ask for help in the repo when you
+open a pr.
+
+If you need to skip a local pre-commit check when you commit changes locally,
+use:
+
+  ```bash
+  git commit "commit message here" --no-verify
+  ```
+
+### About the pre-commit CI Bot
+
+The [Pre-commit CI](https://pre-commit.ci/) bot is a continuous integration service that automatically
+runs pre-commit hooks on each pull request. It also allows you to call the bot
+to fix any issues with the pull request (with the exception of misspelled words).
+
+This helps maintain code quality
+and consistency without requiring contributors to run pre-commit locally.
+
+[Learn more about pre-commit ci in our ci section here](pre-commit-ci)
+
+### Acknowledging New Contributors
+
+We use the [All-contributors bot](https://allcontributors.org/) to acknowledge
+contributors across all of our GitHub repositories. Contributor information
+is then published on [our website here](https://www.pyopensci.org/our-community/index.html#pyopensci-community-contributors).
+
+### Who we acknowledge
+pyOpenSci is liberal in acknowledging all contributions large to small. When
+someone new submits an issue or PR, add them as a contributor to the repo using
+the bot.
+
+* It is ok if their name is already on our website as a contributor,
+if this is their first contribution to a different repo, or a different type of contribution, add them. We will acknowledge the different types of contributions that our wonderful volunteers make.
+* If they are already added as a contributor to the repo where the pull request is, that is ok too. The bot will inform you!
+
+To add a contributor - at the bottom of a pull request or an issue - call the
+following command:
+
+`@all-contributors please add @kiersi for code, review`
+
+Replaceing `@kiersi` with `@contributors-username`.
+
+When you add a user, the bot will open a pull request that can be squashed
+and merged. Once merged, their profile image and name will appear in the
+README file of that repository.
+
+
+pyOpenSci then has an automated build that will parse contributors across all of
+our repos, adding them to
+the contributor list and noting the type(s) of contributions that they have made
+(e.g., packaging guide, peer review guide).
+
+:::{todo}
+## GitHub & CI
+
+* Ci (Continuous integration)  will be run on each new commit added to all of our public repositories (that have content).
+
+
+### Permissions to run CI
+
+In general, things are setup so CI doesn’t run automagically for new contributors. Rather, someone with triage (?) rights will need to approve the workflow to run by hitting the approve and run button that will appear (see image below). We have things setup this way to avoid spam pr’s commits that will abuse our CI use. We could potentially adjust in the future - but also we could adjust but allowing more people with “access levels” that can approve CI.
```

Comment:
```suggestion
In general, things are setup so CI doesn’t run automagically for new contributors. Rather, someone with triage rights will need to approve the workflow to run by hitting the approve and run button that will appear (see image below). We have things set up this way to avoid spam PRs and commits that will abuse our CI use. We could potentially adjust in the future, but also can adjust by allowing more people with “access levels” that can approve CI.
```